### PR TITLE
Update 92-disc-v5-research.md

### PR DESCRIPTION
**Pull request by adambabik**

### CODE REVIEW

**Comment by adambabik** [comment]
@oskarth I agree and we will talk about the next goals today. I will update the PR after collecting information.

File: ideas/92-disc-v5-research.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: 92-disc-v5-research
 title: Research Ethereum Discovery V5 Protocol
-status: Limbo
+status: In Progress
```
Comment (oskarth) [inline ideas/92-disc-v5-research.md]: Active

(sorry, this is confusing...)

File: ideas/92-disc-v5-research.md [hunk 2]
```diff
@@ -80,7 +80,25 @@ Description:
...
-#### Exit criteria
+### Iteration 1
+
+Goal Date: 2018-05-18
+
+Description:
+- [ ] increase awarness by changing the UI copy that Rinkeby network uses Discovery protocol,
+- [ ] make LES working in Mainnet and Ropsten,
+- [ ] design how mailservers can work in dynamic environment,
+- [ ] make it configurable from the UI/command line to use discovery protocol,
```
Comment (dshulyak) [inline ideas/92-disc-v5-research.md]: just to recap. i still think that this is useless. because we either use discovery and stop maintaining list of static peers or we don't and we keep maintaining list of static peers. 

doing both doesn't make sense, as soon as list of static peers will become invalid such fallback option won't work but if we keep it valid all the time - then discovery is not needed.

File: ideas/92-disc-v5-research.md [hunk 3]
```diff
@@ -80,7 +80,25 @@ Description:
...
 1. There is an automated way to collect CPU and network usage metrics in an isolated environment.
 
-#### Exit criteria
+### Iteration 1
+
+Goal Date: 2018-05-18
+
+Description:
+- [ ] increase awarness by changing the UI copy that Rinkeby network uses Discovery protocol,
+- [ ] make LES working in Mainnet and Ropsten,
```
[the remark this answers is not in the document]
Comment (dshulyak) [reply ideas/92-disc-v5-research.md]: maybe i am wrong but i think that ULC requires trusted peers to be hardcoded, so using discovery for les won't make any sense with ulc

[the remark this answers is not in the document]
Comment (adambabik) [reply ideas/92-disc-v5-research.md]: Right, we can postpone it for now because we don't know yet which path we will go (ULC vs LES).

File: ideas/92-disc-v5-research.md [hunk 4]
```diff
@@ -80,7 +80,25 @@ Description:
...
+- [ ] design how mailservers can work in dynamic environment,
```
[the remark this answers is not in the document]
Comment (adambabik) [reply ideas/92-disc-v5-research.md]: Correct. This is just about putting down how mailservers will work with Discovery V5. We can state here that it won't be different from the static solution for now.

File: ideas/92-disc-v5-research.md [hunk 5]
```diff
@@ -80,7 +80,25 @@ Description:
...
+- [ ] make it configurable from the UI/command line to use discovery protocol,
+- [ ] run automated, nightly tests that collect CPU, mem and network I/O statistics with comparison to static nodes.
```
[the remark this answers is not in the document]
Comment (adambabik) [reply ideas/92-disc-v5-research.md]: I was thinking about running your tests daily in Jenkins that were outputting network ingress/egress.

File: ideas/92-disc-v5-research.md (hunk 2 above)
Comment (dshulyak) [reply ideas/92-disc-v5-research.md]: > Our last session proved that it works but at the same time my battery was running down like crazy and there are tons of other edge cases that we need to test and feel comfortable with before we switch it in nightly and alpha releases.

pow was 0.05. i mentioned it during session
